lib/JSON/Validator/OpenAPI.pm

```diff
@@ -10,6 +11,28 @@ my %COLLECTION_RE = (pipes => qr{\|}, csv => qr{,}, ssv => qr{\s}, tsv => qr{\t}
 
 has _json_validator => sub { state $v = JSON::Validator->new; };
 
+{    # proxy methods for compatibility
+  my @proxy_methods = qw<
+    _get_request_uploads
+    _get_request_data
+    _set_request_data
+    _get_response_data
+    _set_response_data
+  >;
+
+  for my $method (@proxy_methods) {
+    no strict 'refs';
+    *{__PACKAGE__ . "::$method"} = sub {
+      deprecated "Using JSON::Validator::OpenAPI directly is DEPRECATED."
+        . " For the Mojolicious-specific methods use JSON::Validator::OpenAPI::Mojolicious";
+
+      shift @_;
+      require JSON::Validator::OpenAPI::Mojolicious;
+      JSON::Validator::OpenAPI::Mojolicious->$method(@_);
```

Comment:
That is what bugged my at the very beginning when I asked how to use JSON::Validator::OpenAPI. The calls to `validate_request` never use `$self`, I think. However the framework-specific methods only need `$c` passed.

I like that you suggest `do` because it's generally discouraged and few people seem to know this style of use. But as these methods work as simple class method I see no reason to change this.